examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
```

Comment:
<p>Q: Sampling of batches should be done randomized or sequential?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Randomization is actually a standard practice and helps with performance at times. But, here since we aren't maximizing performance, I went with that. But, yeah I don't mind implementing it since its a not much of an effort.

Reply:
I won't do this since it would involve steps making the code less clean and the performance we are getting is decent enough.